include/UIKit/UIView.h

```diff
@@ -379,11 +379,10 @@ UIKIT_EXPORT_CLASS
 @end
 
 // WinObjC Xaml Extensions
-@class WXFrameworkElement;
 @interface UIView (UIKitXamlExtensions)
-+ (WXFrameworkElement*)createXamlElement;
-@property (nonatomic, readonly, strong) WXFrameworkElement* xamlElement;
-- (instancetype)initWithFrame:(CGRect)frame xamlElement:(WXFrameworkElement*)xamlElement;
++ (RTObject*)createXamlElement;
+@property (nonatomic, readonly, strong) RTObject* xamlElement;
```

Comment:
that's a great point; can you please add a small comment that calls this out, and demonstrates how to rt_dynamic_cast to the desired projected type?

---
In reply to: [109249693](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109249693) [](ancestors = 109249693)